netlify.toml

```diff
@@ -32,10 +32,18 @@
     # generate the website which is served at /archive/<version>/
     jekyll build --config _config.yml,$(pwd)/netlify/_config_noindex.yml --baseurl /archive/$RELEASE_VERSION --destination _site/archive/$RELEASE_VERSION
 
-    mv _site/sitemap.xml _site/release-legacy-sitemap.xml
-    mv _site/sitemap.xml _site/latest-sitemap.xml
-    mv netlify/sitemap-index.xml _site/sitemap.xml
-    mv netlify/_redirects _site/_redirects
+    # keep the production build if it's not docs.projectcalico.org (off of v3.21 branch) site
+    # REDIRECT_TO_LATEST is set in Netlify UI for environment-variables ONLY for docs.projectcalico.org site.
```

Comment:
Curious about what `REDIRECT_TO_LATEST` means and why it's only in v3.21 (and older?)?

Reply:
The variable `REDIRECT_TO_LATEST` is added in netlify site settings. We add this to docs.projectcalico.org site on netlify. When present, it basically adds the `_redirects_v3.21` to the site. Which have the proxy rules and 301 redirect to new site. This way we ensure that  `docs.projectcalico.org` site proxies manifests and redirects all others